### PR TITLE
Add innodbstats and slavenotifications for mysql

### PR DESCRIPTION
**Pull request by t-8ch**

### CODE REVIEW

**Comment by jlory** [comment]
You should check if innodbstats & slavenotifications  variable are set to true before appending to the template otherwise people using Collectd < 5.5 will have InnodbStats = false in their config file.

**Comment by t-8ch** [comment]
@jlory I made it completely backwards compatible.

**Comment by jlory** [comment]
@t-8ch Awesome, than you!

File: templates/mysql-database.conf.erb [hunk 1]
```diff
@@ -15,5 +15,11 @@
 <%- if @socket -%>
     Socket "<%= @socket %>"
 <%- end -%>
+<%- if not @innodbstats.nil? -%>
+    InnodbStats <%= @innodbstats %>
+<%- end -%>
+<%- if not @slavenotifications.nil? -%>
```
Comment (blkperl) [inline templates/mysql-database.conf.erb]: You can just do `if @slavenotifications`

File: templates/mysql-database.conf.erb [hunk 2]
```diff
@@ -15,5 +15,11 @@
 <%- if @socket -%>
     Socket "<%= @socket %>"
 <%- end -%>
+<%- if not @innodbstats.nil? -%>
```
Comment (blkperl) [inline templates/mysql-database.conf.erb]: You can do `if @innodbstats`  instead

File: templates/mysql-database.conf.erb (hunk 1 above)
Comment (t-8ch) [inline templates/mysql-database.conf.erb]: I wanted to preserve an explicit `slavenotification = false`

**Comment by blkperl** [comment]
Thanks for the contribution!

**Pull request merged**